### ISSUE TITLE
Linker cannot find the CocoaLumberjack library on OSX platform.

### ISSUE THREAD

**Issue by derjohng** [CocoaLumberjack/CocoaLumberjack#870]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | osx 
 Platform Version        | 10.10
 CocoaLumberjack Version | 3.2.0
 Integration Method      |  cocoapods 
 Xcode Version           |  Xcode 8.3.2
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

I always got the error as below.
```
ld: warning: directory not found for option '-L/Users/derjohng/Library/Developer/Xcode/DerivedData/DoItWell-agbyanuvnathgbdbbdtdnxhorfgd/Build/Products/Debug/CocoaLumberjack-OSX10.10'
ld: library not found for -lCocoaLumberjack-OSX10.10
```

I found only **Debug-iphoneos** directory created  under Xcode DerivedData directory 
```./DoItWell-agbyanuvnathgbdbbdtdnxhorfgd/Build/Products/Debug-iphoneos/CocoaLumberjack-OSX10.10```

However, the **Debug** directory is not created.
``./DoItWell-agbyanuvnathgbdbbdtdnxhorfgd/Build/Products/Debug/CocoaLumberjack-OSX10.10``

I do not know why. Because there is not other error message.

**Comment by derjohng** [comment]
Sorry. I forget change  the  Base SDK of the Pod to macos.
